azdev/utilities/command.py

```diff
@@ -60,8 +60,8 @@ def cmd(command, message=False, show_stderr=True, raise_error=False, **kwargs):
     try:
         output = subprocess.check_output(
             command.split(),
+            shell=IS_WINDOWS and command.startswith('az '),
```

Comment:
Even on Windows, `shell=True` should be avoided. 

Consider using `az.bat` instead:

```
> Get-Command az | Format-List

Name            : az.bat
CommandType     : Application
Definition      : D:\cli\py312\Scripts\az.bat
Extension       : .bat
Path            : D:\cli\py312\Scripts\az.bat
FileVersionInfo : File:             D:\cli\py312\Scripts\az.bat
```

Reply:
We can call `shlex.split()` on `command` first. If the first element is `az` and `IS_WINDOWS` is `True`, replace it with `az.bat`.